MIT.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2022 Andisiwe
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```

Comment:
- [ ] Great job adding MIT.md file 💯💡, but I noticed that the link to your MIT in your README file is broken, kindly fix it by changing the path in your README file to `[MIT](./MIT.md)` instead of `[MIT](./LICENSE)` 👌
> OR you can rename this file to `LICENSE` instead of `MIT.md` 💡

![image](https://user-images.githubusercontent.com/45231276/192227920-d87497f3-ecae-4ac9-bffe-38980f5dbd82.png)